plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -72,24 +72,10 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     @Test
     public void testTopNPushdownDisabled()
     {
-        // If topN pushdown gets enabled by default, this test should use a session with topN pushdown disabled
-        assertThat(query("SELECT orderkey FROM orders ORDER BY orderkey LIMIT 10"))
-                .ordered()
-                .isNotFullyPushedDown(TopNNode.class);
-    }
-
-    /**
-     * Verify !SUPPORTS_TOPN_PUSHDOWN declaration is true.
-     */
-    @Test
-    public void verifySupportsTopNPushdownDeclaration()
```

Comment:
Removed this because with the default session having topN enabled this check can be merged with the one guarding `testTopNPushdown`.